themes/scss/widgets/overlays.scss

```diff
@@ -102,3 +98,33 @@
   z-index: 1;
 }
 
+// loader overlay 
+
+.CDB-Loader {
+  @include css3-prefix(animation, loader-progress 1000ms linear 1);
+  @include opacity(0);
+  @include transition(opacity, 1000ms);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 2px;
+  z-index: 10;
```

Comment:
Properties should be ordered position, top, left, width, height, background, z-index